hack/import-flux2-docs.sh

```diff
@@ -8,7 +8,7 @@ controller_version() {
     echo "Please install 'jq'."
     exit 1
   fi
-  curl -s "https://registry.hub.docker.com/v2/repositories/fluxcd/$1/tags/?page_size=1" | jq -r .results[].name
+  curl -s "https://api.github.com/repos/fluxcd/$1/releases" | jq -r '.[] | .tag_name' | sort -V | tail -n 1
```

Comment:
This will give unexpected results with prereleases, see:

```console
$ printf '%s\n' v1.2.0 v.1.1.0 v.1.2.0-alpha v1.3.0-test v2.0.0-alpha | sort -V | tail -n 1
v.1.2.0-alpha
```